modules/rgbd/include/opencv2/rgbd.hpp

```diff
@@ -81,7 +81,7 @@ namespace rgbd
     return (depth != std::numeric_limits<unsigned short int>::min())
         && (depth != std::numeric_limits<unsigned short int>::max());
   }
-  CV_EXPORTS
```

Comment:
`isValidDepth3` is a strange name, it's better to use something like `isValidDepthInt` to make code readable.